index.html

```diff
@@ -1,17 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID" crossorigin="anonymous"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){window.dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-HQREYRLMDN');
+    </script>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@400;700&display=swap" rel="stylesheet" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js" crossorigin="anonymous"></script>
     <title>Awesome Books</title>
 </head>
+
 <body>
+    <nav class="navContainer">
+        <h1 class="title1"><span>Awesome </span> Books</h1>
+        <img src="images/nav.svg" alt="navicon" class="navImg d-none">
+        <img src="images/closeNav.svg" alt="closeNav" class="closeImg d-none">
+        <ul class="navUl">
+            <li id="bookList" class="navLi pointer">Book list</li>
+            <li id="addNew" class="navLi pointer">Add New</li>
+            <li id="contactInfo" class="navLi pointer">Contact info</li>
+        </ul>
+    </nav>
```

Comment:
- [ ] To enhance the user experience of this application kindly ensure that an **active menu link is highlighted**(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your nav menu:**
![image](https://user-images.githubusercontent.com/12745474/138341539-3a61dbde-4275-48dc-8d7f-a521f3487278.png)


**The wireframe nav menu:**
![image](https://user-images.githubusercontent.com/12745474/138342596-c4a68582-914a-4c5a-b255-8db0325cbaca.png)